docs/logging/logging.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 1
+sidebar_label: Logging
+title: ""
+keywords:
+- Harvester
+- harvester
+- Logging
+---
+
+# Harvester Logging
+
+_Available as of v1.1.0_
+
+The Harvester logging infrastructure allows you to aggregate Harvester logs into an external service such as [Elasticsearch](https://www.elastic.co/elasticsearch/),
+[Grafana Loki](https://grafana.com/oss/loki/), or [Splunk](https://www.splunk.com/).
+
+## Collected Logs
+
+While users are able to configure and modify where the aggregated logs are sent, they cannot change which logs are
+aggregated. See below for a list of which logs are collected:
```

Comment:
```suggestion
While users are able to configure and modify where the aggregated logs are sent, they cannot change which logs are aggregated. See below for a list of logs that are collected:
```

Reply:
maybe `See below for a list of logs that are collected:` is easier to understand?

Reply:
I also combined the two lines into one as they are one single sentence~